### ISSUE TITLE
Next,Previous and seekbar not working

### ISSUE THREAD

**Issue by wasbt** [jeancsanchez/JCPlayer#68]
Hi @jeancsanchez thank you for your solution,
When i tap in the area of seekbar, next and previous button,seekbar doesnt work  and  spinner is showing
Version 2.6.11
https://imgur.com/a/WezUauh

**Comment by wasbt** [comment]
I was able to fix it 
```
        jcplayerView = (JcPlayerView) findViewById(R.id.jcplayer);
        jcplayerView.initPlaylist(jcAudios, this);

```


```
  @Override
    public void onTimeChanged(@NonNull JcStatus status) {
        if(!seekbar_jcplayer.isEnabled())
            seekbar_jcplayer.setEnabled(true);

    }
```

**Comment by peace-shillong** [comment]
@wassimrel  
Where Did you keep this line of code 
 @Override
    public void onTimeChanged(@NonNull JcStatus status) {
        if(!seekbar_jcplayer.isEnabled())
            seekbar_jcplayer.setEnabled(true);

    }

I tried to keep it in the MainActivity of the Sample File it shows undefined variable : seekbar_jcplayer.

**Comment by peace-shillong** [comment]
@jeancsanchez We have been facing this Problem Since v1 and I've tested v2 also and i don't know why in the new version I can use the seekbar in all the songs but when I use the seekbar, I'm unable to use All the buttons and the seekbar also.
![screenshot_20180826-224451](https://user-images.githubusercontent.com/33800152/44630924-85c62c80-a982-11e8-945b-c046fd70572c.png)
This is what i get on pressing the seekbar

This is what i see hope you see the screen shot

**Comment by basitsaleemmb** [comment]
@wassimrel 
!seekbar_jcplayer is not defined

**Comment by wasbt** [comment]
seekbar_jcplayer = findViewById(com.example.jean.jcplayer.R.id.seekBar);

**Comment by basitsaleemmb** [comment]
One thing more after pausing I did not get play button button.

On Sep 3, 2018 6:44 PM, "wassimrel" <notifications@github.com> wrote:

seekbar_jcplayer = findViewById(com.example.jean.jcplayer.R.id.seekBar);

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/jeancsanchez/JcPlayer/issues/68#issuecomment-418119907>,
or mute the thread

**Comment by jeancsanchez** [comment]
@basitsaleemmb @wassimrel @peace-shillong I did a paliative fix on version 2.6.12.

**Comment by Moseqi** [comment]
It is still happening.
